content/en/docs/ops/setup/standalone-operator/index.md

```diff
@@ -1,196 +1,141 @@
-    istio-grafana-post-install-release-1.1-20190308-09-16-2p7m5    0/1     Completed   0          2m
-    istio-ingressgateway-7ddcfd665c-dmtqz                          1/1     Running     0          1m
-    istio-pilot-f479bbf5c-qwr28                                    2/2     Running     0          1m
-    istio-policy-6fccc5c868-xhblv                                  2/2     Running     2          1m
-    istio-security-post-install-release-1.1-20190308-09-16-bmfs4   0/1     Completed   0          2m
-    istio-sidecar-injector-78499d85b8-x44m6                        1/1     Running     0          1m
-    istio-telemetry-78b96c6cb6-ldm9q                               2/2     Running     2          1m
-    istio-tracing-69b5f778b7-s2zvw                                 1/1     Running     0          1m
-    kiali-99f7467dc-6rvwp                                          1/1     Running     0          1m
-    prometheus-67cdb66cbb-9w2hm                                    1/1     Running     0          1m
-    {{< /text >}}
-
-## Deploy your application
-
-You can now deploy your own application or one of the sample applications
-provided with the installation like [Bookinfo](/docs/examples/bookinfo/).
-
-{{< warning >}}
-The application must use either the HTTP/1.1 or HTTP/2.0 protocols for all its HTTP
-traffic; HTTP/1.0 is not supported.
-{{< /warning >}}
-
-When you deploy your application using `kubectl apply`,
-the [Istio sidecar injector](/docs/setup/additional-setup/sidecar-injection/#automatic-sidecar-injection)
-will automatically inject Envoy containers into your
-application pods if they are started in namespaces labeled with `istio-injection=enabled`:
+You can also enable or disable specific features or components.
+For example, to disable the telemetry feature:
 
 {{< text bash >}}
-$ kubectl label namespace <namespace> istio-injection=enabled
-$ kubectl create -n <namespace> -f <your-app-spec>.yaml
+$ kubectl apply -f - <<EOF
+apiVersion: install.istio.io/v1alpha2
+kind: IstioControlPlane
+metadata:
+  namespace: istio-operator
+  name: example-istiocontrolplane
+spec:
+  profile: default
+  telemetry:
+    enabled: false
+EOF
 {{< /text >}}
 
-In namespaces without the `istio-injection` label, you can use
-[`istioctl kube-inject`](/docs/reference/commands/istioctl/#istioctl-kube-inject)
-to manually inject Envoy containers in your application pods before deploying
-them:
-
-{{< text bash >}}
-$ istioctl kube-inject -f <your-app-spec>.yaml | kubectl apply -f -
-{{< /text >}}
+Refer to the [`IstioControlPlane` API](/docs/reference/config/istio.operator.v1alpha12.pb/)
+for the complete set of configuration settings.
 
 ## Uninstall
 
-Delete the Istio Operator and Istio deployment:
+Delete the Istio operator and Istio deployment:
 
 {{< text bash >}}
 $ kubectl -n istio-operator get IstioControlPlane example-istiocontrolplane -o=json | jq '.metadata.finalizers = null' | kubectl delete -f -
```

Comment:
These uninstall instructions need a change.  They represent the early implementaton of the oprator which had a broken removal requiring a force and removal of the finalizers metadata.